src/Symfony/Component/Console/Tests/Command/CommandTest.php

```diff
@@ -328,6 +328,11 @@ public function testRunWithProcessTitle()
         $command->setProcessTitle('foo');
         $this->assertSame(0, $command->run(new StringInput(''), new NullOutput()));
         if (function_exists('cli_set_process_title')) {
+            if (null === @cli_get_process_title()) {
+                if ('Darwin' === PHP_OS) {
```

Comment:
The idea was to skip the test only if you're on OS X. If it fails on another platform, it should not be skipped (at least until we know why and there isn't any workaround)